src/js/utils/JSONUtil.js

```diff
@@ -0,0 +1,333 @@
+/**
```

Comment:
@wavesoft did you grab some of this code from somewhere? The spacing all over this file is inconsistent with the rest of our codebase.